tests/security/cc/audit_remote_libvirt.pm

```diff
@@ -0,0 +1,66 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+#
+# Summary: Run 'audit-remote-libvirt' test case of 'audit-test' test suite
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#96531
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use audit_test qw(run_testcase compare_run_log);
+
+sub run {
+    my ($self) = shift;
+
+    my $image_path = '/var/lib/libvirt/images';
+    select_console 'root-console';
+
+    # Install the required packages for libvirt environment setup,
+    zypper_call("in qemu libvirt virt-install virt-manager");
+
+    # Start libvirtd daemon and start the default libvirt network
+    assert_script_run("systemctl start libvirtd");
+    assert_script_run("virsh net-start default");
+    assert_script_run("systemctl is-active libvirtd");
+    assert_script_run("virsh net-list | grep default | grep active");
+
+    # Download the pre-installed guest images and sample xml files
+    my $vm_name      = 'vm-swtpm-legacy';
```

Comment:
IMHO, If the `$vm_name` could change to something about `audit-remote`  will be better.

Reply:
> IMHO, If the `$vm_name` could change to something about `audit-remote` will be better.

You are right, I have thought this but I reused test case 'swtpm' data files if I change the vm name I have to change the data files' contents accordingly (the code change is not so simple).
So I decided not to do that.